girder/api/v1/folder.py

```diff
@@ -20,6 +20,7 @@ def __init__(self):
         self.route('DELETE', (':id',), self.deleteFolder)
         self.route('DELETE', (':id', 'contents'), self.deleteContents)
         self.route('GET', (), self.find)
+        self.route('GET', ('position', ':id'), self.findPosition)
```

Comment:
To be consistent with the other route names, this should be `(':id', 'position')`

Reply:
My rationale was that the `item/:id` endpoints operate on a specific item, but this endpoint was an accessory to the find endpoint and, therefore, `position` was more significant than `:id`.  If you aren't convinced by that rationale, I'll change the order.

Reply:
I prefer `:id/position`; it gives the position of the folder given by `:id`, in the same way that `:id/details` gives the details of the folder given by `:id`.